NEWS.md

```diff
@@ -320,7 +323,7 @@ https://cvc5.github.io
 https://cvc5.github.io/docs
 
 ## System Description
-**cvc5: A Versatile and Industrial-Strength SMT Solver.**  
+**cvc5: A Versatile and Industrial-Strength SMT Solver.**
```

Comment:
We need the spaces here to force a line break.

Reply:
I see. Reverted.